drivers/hmis/app/models/hmis/search/assessment_search.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Search::AssessmentSearch
+  attr_accessor :input
+  def initialize(input)
+    self.input = input
+  end
+
+  def results(scope)
```

Comment:
```suggestion
  def apply_filters(scope)
```
maybe? "results" was a bit confusing to me.

Reply:
Oh I see, we use `apply_filters` elsewhere. Maybe just `filter` or `filter_scope`?